Libraries/LibGUI/Model.cpp

```diff
@@ -57,10 +59,6 @@ void Model::did_update(unsigned flags)
 {
     for (auto* client : m_clients)
         client->model_did_update(flags);
-
-    for_each_view([&](auto& view) {
-        view.did_update_model(flags);
-    });
```

Comment:
I just want to say that I love this change.

Reply:
Me too!